### PR TITLE
update the roadmap

### PR DESCRIPTION
**Pull request by b00f**
Updates the roadmap

### CODE REVIEW

File: website/_data/roadmap.yml [hunk 1]
```diff
@@ -1,37 +1,51 @@
...
+- status: upcoming
+  title: Pactus Virtual Machine (PVM)
+  date: Q3 2025
+
+- status: upcoming
+  title: Mobile Friendly GUI
+  date: Q2 2025
+
+- status: upcoming
+  title: Decentralized Storage Platform
```
Comment (kehiy) [inline website/_data/roadmap.yml:26]: What we mean by `Decentralized Storage Platform` we are not aiming to be a storage layer like filecoin or chia coin...

File: website/_data/roadmap.yml [hunk 2]
```diff
@@ -1,37 +1,51 @@
...
+  title: Decentralized Password Manger
+  date: Q4 2025
 
 - status: upcoming
   title: Browser Extension
-  date: Q1 2025
+  date: Q3 2025
+
+- status: upcoming
+  title: Pactus Virtual Machine (PVM)
```
Comment (kehiy) [inline website/_data/roadmap.yml:18]: PVM is already used in Polkadot, we may need a new name. also, We can use Pactus beside the VM when we designed the VM ourselves I believe.

File: website/_data/roadmap.yml (hunk 1 above)
Comment (b00f) [reply website/_data/roadmap.yml:26]: It is one of the goal of Pactus to let user purchase storage on Pactus.

Comment (kehiy) [reply website/_data/roadmap.yml:26]: I think it needs clarifications and discussion before adding to the roadmap IMHO.

File: website/_data/roadmap.yml (hunk 2 above)
Comment (kehiy) [reply website/_data/roadmap.yml:18]: OK, we discuss for a name, lets keep this name for now.

File: website/_data/roadmap.yml (hunk 1 above)
Comment (b00f) [reply website/_data/roadmap.yml:26]: OK, Let's talk about it in the next Dev-meeting.

Comment (kehiy) [reply website/_data/roadmap.yml:26]: Good idea, I agree.

**Pull request merged**